### DISCUSSION TITLE
Resume Bundle X ATS Scoring

### DISCUSSION THREAD

**Discussion by harikanani** [recodehive/Stackoverflow-Analysis#348] [Q&A] [answered]
[Harikrushn_Kanani_Backend_Developer_Resume.pdf](https://github.com/user-attachments/files/16323894/Harikrushn_Kanani_Backend_Developer_Resume.pdf)

**Comment by sanjay-kv** [comment]
[ATS Version of CV (8).docx](https://github.com/user-attachments/files/16324018/ATS.Version.of.CV.8.docx)
ATS version of your resume

**Comment by sanjay-kv** [comment]
Lets have further discussion here

**Reply by matiborda** [reply, marked as answer]
okay

**Comment by sanjay-kv** [comment]
[review hari.pdf](https://github.com/user-attachments/files/16324022/review.hari.pdf)
Reviewed Resume

**Comment by vijaychandra1910** [comment]
I want to get galaxy  brain batch please help.

**Reply by sanjay-kv** [reply]
https://www.youtube.com/watch?v=sbyXpflAXkQ&list=PLrLTYhoDFx-kiuFiGQqVpYYZ56pIhUW63&index=10

i made complete videos on it please watch it here

**Comment by Yanzz231** [comment]
a

**Comment by DarkHacker28** [comment]
Use. :


https://resumeworded.com/resume-scanner


and good templete
